### ISSUE TITLE
Question: Does it work, if files are refactored?

### ISSUE THREAD

**Issue by schittli** [beansoft/iscratch-support#15]
Hello
Unfortunately, the description does not mention whether the plugin can also handle code refactoring. I ask because it looks like the Note saves a link to the file and line number, e.g. `file://c:/…hello.java#130`.

What happen, for example, if:
* The project is moved from 
 `c:\source\opensource\project1` to 
 `c:\source\archive\WebSite Rel 1`?
* Source files are deleted?
* Source files are renamed?
* If `void myFunc() {…}` has a note and then this function is moved from line 30 to line 900?

Does the Note then lose the link to the correct source code?

Thanks a lot, kind regards,
Thomas

**Comment by beansoft** [comment]
Yes, it's not supported so far, it only links to lines. And some customer also reported this issue. And I've found IDEA's bookmark could move correctly between lines. I'll dig into IDEA's source code to see if I could improve this feature.
